kong/dao/schemas/upstreams.lua

```diff
@@ -189,6 +191,20 @@ return {
       -- header name, if `hash_fallback == "header"`
       type = "string",
     },
+    hash_on_cookie = {
+      -- cookie name, if `hash_on == "cookie"`
+      type = "string",
+    },
+    hash_on_cookie_path = {
+      -- cookie name, if `hash_on == "cookie"`
+      type = "string",
+      default = "/",
+    },
+    hash_fallback_cookie_path = {
```

Comment:
Should this not support falling back to a cookie value if the primary `hash_on` is `header` or `ip`? I can't think of any use cases for it, but currently `header` falling back on `ip` and vice-versa are supported.

Reply:
yes it should. But with headers we can have 2 different headers, one for primary, one for fallback. With cookies, there is never more than one since the cookie is created when absent.

Hence having `hash_on_cookie` and `hash_on_cookie_path` is enough. We do not need `hash_fallback_cookie_path` (actually in the current state, `hash_fallback_cookie` is missing).